website/docs/cloud-docs/api-docs/policy-set-params.mdx

```diff
@@ -36,7 +36,9 @@ page_title: Policy Set Parameters - API Docs - Terraform Cloud
 
 # Policy Set Parameters API
 
--> **Note:** Sentinel and OPA policies are paid features, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+-> **Note:** Sentinel and OPA policies available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta.
+
+Policy Set Parameters allow the ability for policies to receive parameters for use within the policy.
 
 This set of APIs covers create, update, list and delete operations on parameters.
```

Comment:
And then you know what would also be helpful? Just like one sentence of example. like "For example, you could create a parameter to do z, y, z."  Sometimes I think stuff like this can help users visualize why they would want to use this API/why it's helpful.

```suggestion
Policy Set Parameters let you inject parameters into an existing policy. This set of APIs covers create, update, list and delete operations on parameters.
```